### ISSUE TITLE
PDF Link loads another page and no pdf generated

### ISSUE THREAD

**Issue by iamlinkus** [iusvar/grav-plugin-snappygrav#6]
Hey, I just added snappygrav to a fresh grav installation, installed both vendor packages as needed, but when I add a link to the page, using the code that's written in the readme, it doesn't generate a pdf, just loads the same/other page with :pdf at the end of the URL and nothing else happens. Can this be incompatible with the most recent grav release?

**Comment by iusvar** [comment]
While I try to solve this problem I have just released a new plugin on printing called PrintFriendly. Should soon be available via Grav Package Manager (GPM)...

**Comment by phillchill** [comment]
**summary:** clicking :pdf link loads other page, whose template throws "Undefined variable: page_slug" exception

I have installed both required libraries via composer.
I encountered the following error when clicking the `:pdf` link included in my `/about` page which is rendered by the `page.html.twig` template:
```
 Twig_Error_Runtime
An exception has been thrown during the rendering of a template ("Undefined variable: page_slug").
```
![screen shot 2017-03-09 at 11 19 18](https://cloud.githubusercontent.com/assets/902958/23745923/5ba7c9e0-04ba-11e7-8329-a1828ab21397.png)

weird thing about this is that it points to a line in another template (`landing.html.twig`)
I removed the block of lines that seemed to throw the error from the `landing.html.twig` template and tried to click the link again. This results in loading of the page using the `landing.html.twig` template.

So my assesment is that there are 2 things going wrong (for me):
1. clicking the pdf link sometimes redirects to another page. (seems like a general error reported here by @iamlinkus )
2. the other page that is mistakenly loaded throws the 'undefined `page_slug` error' (similar to #10 ) (seems like something specific to template structure, so might be harder to reproduce)
